src/Compilers/VisualBasic/Portable/Declarations/MergedTypeDeclaration.vb

```diff
@@ -64,14 +64,21 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             End Get
         End Property
 
-        Public Function GetAttributeDeclarations() As ImmutableArray(Of SyntaxList(Of AttributeListSyntax))
+        Public Function GetAttributeDeclarations(Optional quickAttributes As QuickAttributes? = Nothing) As ImmutableArray(Of SyntaxList(Of AttributeListSyntax))
```

Comment:
nit: seems odd that the VB version uses an optional parameter while the C# version uses overloading.

Reply:
yes.  def.  i will unify that.

Reply:
made both into optional parameters.